_layouts/default.html

```diff
@@ -113,7 +116,10 @@
           <a href="{{ site.baseurl }}/blinky.html">3. blinky</a>
         </li>
         <li>
-          <a href="{{ site.baseurl }}/modules.html">4. modules<!--ul-->
+          <a href="{{ site.baseurl }}/ap.html">4. access point</a>
```

Comment:
That makes sense. I wasn't too sure about the placement but I knew it would be nice to put it after blinky, so people can learn just about the LEDs first.